script.js

```diff
@@ -12,3 +12,359 @@ mobileLinks.addEventListener('click', () => {
   title.style.display = 'block';
   mobileLinks.classList.toggle('open');
 });
+
+/* grid first article */
+function createLinks(array) {
+  let list = '';
+  array.forEach((item) => {
+    list += `<li> <a  class="is-link" href="#">${item}</a></li>`;
+  });
+  return list;
+}
+const articleOne = {
+  sectionTitle: 'My Recent Works',
+  sectionImage: {
+    src: 'assets/ImgPlaceholder@2x.png',
+    alt: 'section image',
+  },
+  articleOneTitle: 'Multi-Post Stories',
+  itemParagraph: `A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.`,
+  technologyA1: ['css', 'html', 'bootstrap', 'Ruby'],
+  projectBtn: 'See Project',
```

Comment:
- [ ] Kindly ensure all your popups are functional currently I can see this project popup is not opening.